src/frequenz/sdk/microgrid/client.py

```diff
@@ -396,6 +422,7 @@ async def _expect_category(
     async def meter_data(
         self,
         component_id: int,
+        timeout: float = DEFAULT_TIMEOUT,
```

Comment:
I think it should be one global default timeout - just to make sure program won't stuck.
Passing timeout as an argument would be problematic in case of channel communication.

Reply:
Why would passing the timeout as an argument be problematic in the context of channels?

It is much easier and convenient to work with it as an argument in tests, because you can just pass it directly, without having to patch it.

Reply:
> Why would passing the timeout as an argument be problematic in the context of channels?

1. Because many actor would subscribe for the same channel.
What should happen if one actor subscribe for `meter_data with timeout = 2` and other actor would subscribe for `meter_data with timeout=60` ?
2. It is breaking change that is not needed now. Doing breaking changes for tests is problematic.

> It is much easier and convenient to work with it as an argument in tests, because you can just pass it directly, without having to patch it.

But you can import that DEFAUTL_TIMEOUT in the tests and override it, yes?

Reply:
Having it as a single value also makes it easy to configure, from the UI for example.

Reply:
Playing devil's advocate, what would be the solution if for some reason you want only one particular call to have a smaller (or bigger) timeout? If we leave this option, the default could be also be still configured via the UI.

Reply:
> 1. Because many actors would subscribe for the same channel.
What should happen if one actor subscribe for meter_data with timeout = 2 and other actor would subscribe for meter_data with timeout=60 ?

Good point. Yeah, in this case, making it global would simplify it. Otherwise, it would have to be yet another constituent of a channel. 

I think there might be cases when actors that subscribe for data that might want to use different timeouts, but probably it will not happen soon.

> But you can import that `DEFAULT_TIMEOUT` in the tests and override it, yes?

I would have to patch it as a module variable, but should be doable.

Ok, so everyone agrees that it the timeout should be implemented as a global variable instead of passed as a param for now? I'll update the code once we make a decision that is the way to go.